tests/integrations/aws_lambda/test_aws.py

```diff
@@ -248,25 +247,27 @@ def test_init_error(run_lambda_function):
             """
         init_sdk()
         func()
+
         def test_handler(event, context):
             return 0
         """
         ),
         b'{"foo": "bar"}',
+        syntax_check=False,
     )
     log_result = (base64.b64decode(response["LogResult"])).decode("utf-8")
     expected_text = "name 'func' is not defined"
     assert expected_text in log_result
```

Comment:
@rhcarvalho @untitaker In this particular test case as well, there is no events data which was coming and thus only comparison was with log_result that we get from response.
Now, I've figured out why this event data is not coming as well. It is because in the initialization error wrapper that we have currently written, it takes the execution info directly from the system (sys.exc_info) and not from an exact exception which is generated. So, we actually need to generate an exception as we only receive data from (sys.exc_info) in the post_init_error method and not the actual exception. A simple solution is reraise the exception from the (sys.exc_info) and then capture that exception information, which solves all the problems. I've tested and verified it. I'll tag you guys at the exact method I'm talking about.